services/consensusalgo/benchmarkconsensus/non_leader.go

```diff
@@ -82,16 +81,21 @@ func (s *service) nonLeaderCommitAndReplyUnderMutex(blockPair *protocol.BlockPai
 		return err
 	}
 
+	// prepare the message
+	message := &gossipmessages.BenchmarkConsensusCommittedMessage{
+		Status: status,
+		Sender: (&gossipmessages.SenderSignatureBuilder{
+			SenderPublicKey: s.config.NodePublicKey(),
+			Signature:       sig,
+		}).Build(),
+	}
+
 	// send committed back to leader via gossip
+	recipient := blockPair.ResultsBlock.BlockProof.BenchmarkConsensus().Sender().SenderPublicKey()
+	s.reporting.Info("Replying committed with last committed height", instrumentation.BlockHeight(s.lastCommittedBlockHeight()), instrumentation.String("signed-data", signedData.String()))
```

Comment:
We have `instrumentation.Bytes` for binary data and `instrumentation.Stringable` for anything implementing a `Stringer` interface.

Reply:
what's the difference between `instrumentation.Stringable` and `instrumentation.String`? isn't it exactly the same thing since they're both doing `String()`?

Reply:
`Stringable` converts something to `String` but `String` itself does not implement a `Stringer`

Reply:
Cool, fixed

https://github.com/orbs-network/orbs-network-go/commit/64909efe9973b1de9a80090d2cf2175c04c0198c